manifests/init.pp

```diff
@@ -67,38 +84,37 @@
   windowsfeature { "${_web_asp_net}":
   } ->
 
-  # remove default web site
-  iis::manage_site {'Default Web Site':
-    ensure    => absent,
-    site_path => 'any',
-    app_pool  => 'DefaultAppPool',
-    require   => Windowsfeature['Web-WebServer'],
-  } ->
-
-  # application in iis
+  # create application pool in iis
   iis::manage_app_pool { "${_chocolatey_server_app_pool_name}":
     enable_32_bit           => true,
     managed_runtime_version => 'v4.0',
   } ->
+
+  # create website for chocolatey server
   iis::manage_site {'chocolatey.server':
     site_path  => $_chocolatey_server_location,
     port       => "${_chocolatey_server_app_port}",
     ip_address => '*',
     app_pool   => "${_chocolatey_server_app_pool_name}",
     require    => Package['chocolatey.server'],
-  } ->
+  }
 
   # lock down web directory
   acl { "${_chocolatey_server_location}":
     purge                      => true,
     inherit_parent_permissions => false,
     permissions                => [
-      { identity => 'Administrators', rights => ['full'] },
-      { identity => 'IIS_IUSRS', rights => ['read'] },
-      { identity => 'IUSR', rights => ['read'] },
-      { identity => "IIS APPPOOL\\${_chocolatey_server_app_pool_name}", rights => ['read'] }
+      { identity => 'Administrators',
+      rights     => ['full'] },
+      { identity => 'IIS_IUSRS',
+      rights     => ['read'] },
+      { identity => 'IUSR',
+      rights     => ['read'] },
+      { identity => "IIS APPPOOL\\${_chocolatey_server_app_pool_name}",
+      rights     => ['read'] }
```

Comment:
Lint hates the way I had it, so thanks for that. I'm much more a fan of it getting it so that lint ignores these lines (not a fan of dropping lines here as it makes it harder to read each permission). No worries though.